src/main/java/mate/acadamy/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,156 @@
+package mate.acadamy.internetshop.dao.jdbc;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.acadamy.internetshop.dao.OrderDao;
+import mate.acadamy.internetshop.dao.lib.Dao;
+import mate.acadamy.internetshop.exceptions.DataProcessingException;
+import mate.acadamy.internetshop.model.Order;
+import mate.acadamy.internetshop.model.Product;
+import mate.acadamy.internetshop.util.ConnectionUtil;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public Order create(Order order) {
+        String query = "INSERT INTO internet_shop.orders (user_id) VALUES(?);";
```

Comment:
```suggestion
        String query = "INSERT INTO orders (user_id) VALUES(?);";
```
Прибери всюди використання назви схеми. Схему ми конфігуруємо при формуванні підключення в ConnectionUtils

Reply:
Done